src/main/index.js

```diff
@@ -230,6 +230,52 @@ const createPrivacyWindow = () => {
     return window;
 };
 
+const createUsbWindow = () => {
+    const window = createWindow({
+        width: 400,
+        height: 300,
+        parent: _windows.main,
+        search: 'route=usb',
+        modal: true,
+        frame: false
+    });
+
+    // Filters from navigator.usb.requestDevice do not appear to be available here.
+    // Hard code to micro:bit since that is the only device that currently uses this api.
+    const getIsMicroBit = device => device.vendorId === 0x0d28 && device.productId === 0x0204;
```

Comment:
I think this is fine for now, but do you know how we'd handle this if we supported more than one kind of device? It didn't seem obvious when I skimmed the API docs you linked. Or, given that the vendor and product ID are provided to `navigator.usb.requestDevice`, I don't suppose devices are getting filtered before the `usb-device-added` event...?

Reply:
No, the devices are not getting filtered. Without this I was seeing all USB devices connected to my computer. I also tried inspecting the event passed to the `'select-usb-device'` handler hoping the filter contents would be in there, but I didn't see it. 

If we really needed to support more than one kind of device, the only thing that comes to mind is somehow having gui communicate that to Electron using Electron APIs. I don't really like that.

When I have some time I'll see if I can find a place to post this question to Electron. Maybe they will add it if it isn't something I'm missing.